update_database.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+require 'functions.php';
+
+$db = connectToDB();
+
+// DESPERATELY in need of some refactoring - but it works!
+if (!empty($_POST['update_title'])) {
```

Comment:
DRY - You could also just do if (count($_POST)===5) would save having to repeat yourself